lib/fs/debug.go

```diff
@@ -14,9 +14,9 @@ import (
 )
 
 var (
-	l = logger.DefaultLogger.NewFacility("filesystem", "Filesystem access")
+	l = logger.DefaultLogger.NewFacility("fs", "Filesystem access")
```

Comment:
This needs to be changed in the help text in main too (and subsequently in docs).

Reply:
Help text in main is generated from available facilities.